pkg/cmd/registry/artifact/metadata/get.go

```diff
@@ -113,5 +117,38 @@ func runGet(opts *GetOptions) error {
 
 	opts.Logger.Info(icon.SuccessPrefix(), opts.localizer.MustLocalize("artifact.common.message.artifact.metadata.fetched"))
 
+	err = printBrowserUrl(opts, &response)
+	if err != nil {
+		return err
+	}
+
 	return dump.Formatted(opts.IO.Out, opts.outputFormat, response)
 }
+
+func printBrowserUrl(opts *GetOptions, metadata *registryinstanceclient.ArtifactMetaData) error {
+	conn, err := opts.Connection(connection.DefaultConfigRequireMasAuth)
+	if err != nil {
+		return err
+	}
+
+	registry, _, err := serviceregistry.GetServiceRegistryByID(opts.Context, conn.API().ServiceRegistryMgmt(), opts.registryID)
+	if err != nil {
+		return err
+	}
+
+	group := metadata.GetGroupId()
+
+	if group == "" {
+		group = util.DefaultArtifactGroup
+	}
+
+	homeURL, ok := registry.GetBrowserUrlOk()
+
+	if ok {
+		registryURL := fmt.Sprintf("%s/artifacts/%s/%s/versions/%s", *homeURL, group, metadata.Id, metadata.Version)
+		opts.Logger.Info(opts.localizer.MustLocalize("artifact.common.webURL", localize.NewEntry("URL", color.Info(registryURL))))
+	}
```

Comment:
This is repeated across 2 commands. IMHO it should be all part of the helper instead of just getting browser url

Reply:
Not including logger part that is not needed.